src/Haskell/Ide/Engine/Plugin/Brittany.hs

```diff
@@ -44,7 +46,8 @@ brittanyCmd tabSize uri range =
                 l = length textLines
                 c = T.length $ last textLines
                 textLines = T.lines text
-                textEdit = J.TextEdit (Range startPos endPos) newText
+                newTextWithoutNewline = T.init newText -- get rid of extraneous \n
```

Comment:
Wouldn't it be a better idea to remove all trailing newlines specifically instead of blindly truncating the text?

Reply:
Probably! Brittany seems to only remove the first trailing newline though, so should we try to match this?